packages/ui5-middleware-cfdestination/test/auth/ui5-auth-in-yaml.yaml

```diff
@@ -0,0 +1,25 @@
+specVersion: "2.2"
+metadata:
+  name: ui5-app
+type: application
+framework:
+  name: OpenUI5
+  version: "1.89.1"
+  libraries:
+    - name: sap.m
+    - name: sap.ui.core
+    - name: sap.ui.layout
+    - name: themelib_sap_fiori_3
+server:
+  customMiddleware:
+  - name: ui5-middleware-cfdestination
+    afterMiddleware: compression
+    configuration:
+      authenticationMethod: "route"
+      debug: true
+      port: 1091
+      xsappJson: "xs-app.json"
+      destinations:
+      # check that the destination name (here: "backend") matches your router in xssppJson
+      - name: "backend"
+        url: "https://services.odata.org/V4/(S(fdng4tbvlxgzpdtpfap2rqss))/TripPinServiceRW/"
```

Comment:
is this your personal token? Is it OK to submit it? I am wondering whether the token can be injected as env param - or will it also mean to have it submitted with GH actions? I remember GH actions allow to inject env vars, right?

Reply:
no, it's a public token for `r/w` on the official OData v4 sample service